apps/music/elements/music-artwork.js

```diff
@@ -236,11 +236,7 @@ proto.createdCallback = function() {
   });
 
   this.onDOMLocalized = () => {
```

Comment:
If you are renaming your event on `document` to "DOMRetranslated", please also rename this callback to `onDOMRetranslated` to match.